javascript/features/gangs/gang_manager.js

```diff
@@ -12,7 +13,7 @@ import MockGangDatabase from 'features/gangs/test/mock_gang_database.js';
 class GangManager {
     constructor(settings) {
         this.database_ = server.isTest() ? new MockGangDatabase()
-                                         : new GangDatabase();
+            : new GangDatabase();
```

Comment:
nit: whitespace

Reply:
Autoformatter doesn't understand the tabs :( 
Fixed 👍